packages/checkout-storefront/src/lib/utils/utils.ts

```diff
@@ -53,7 +53,7 @@ export const clearUrlAfterPasswordReset = (): void => {
 };
 
 export const replaceUrl = ({
-  url = window.location.origin,
+  url = window.location.origin + window.location.pathname,
```

Comment:
I think we can change this to `window.location.href` or `window.location.toString()`
```suggestion
  url = window.location.toString(),
```